test/Microsoft.AspNetCore.Blazor.Build.Test/RuntimeDependenciesResolverTest.cs

```diff
@@ -67,17 +67,30 @@ public void FindsReferenceAssemblyGraph_ForStandaloneApp()
                 "mscorlib.dll",
                 "netstandard.dll",
                 "StandaloneApp.dll",
+                "System.dll",
+                "System.Collections.Concurrent.dll",
+                "System.Collections.dll",
                 "System.ComponentModel.Composition.dll",
+                "System.ComponentModel.dll",
                 "System.Core.dll",
                 "System.Data.dll",
-                "System.dll",
+                "System.Diagnostics.Debug.dll",
                 "System.Drawing.dll",
                 "System.IO.Compression.dll",
                 "System.IO.Compression.FileSystem.dll",
+                "System.Linq.dll",
+                "System.Linq.Expressions.dll",
                 "System.Net.Http.dll",
                 "System.Numerics.dll",
+                "System.Reflection.Emit.ILGeneration.dll",
+                "System.Reflection.Emit.Lightweight.dll",
+                "System.Reflection.Primitives.dll",
+                "System.Resources.ResourceManager.dll",
+                "System.Runtime.dll",
+                "System.Runtime.Extensions.dll",
                 "System.Runtime.Serialization.dll",
                 "System.ServiceModel.Internals.dll",
+                "System.Threading.dll",
```

Comment:
The set of references changed with netcoreapp2.1 - should we be concerned about this?

Reply:
I'd like to know what the set of dependencies the default project ends up with after the linker runs.

 * Is it much bigger?
 * Does it try to use `System.Reflection.Emit.*.dll` for anything at runtime (which would be very slow on Mono WebAssembly, and won't work at all with AoT)? I guess this would be related to the ASP.NET change to DI that does some IL generation.

If it's not significantly bigger, and especially if it links out `System.Reflection.Emit.*.dll` completely, then I'm sure it would be fine.

If it is significantly bigger it raises more difficult questions.